.gitignore

```diff
@@ -62,3 +62,9 @@ buck-out/
 *.jsbundle
 electron/
 public/favicon*
+
+# Spacemacs
+**/.#*
```

Comment:
The flycheck plugin for spacemacs drops a bunch of turds every time it analyzes a file. This just excludes those.